classic/src/view/container/Viewport.js

```diff
@@ -1,6 +1,12 @@
 Ext.define('momo.view.container.Viewport', {
     extend: 'Ext.container.Viewport',
 
+    requires: [
+               'BasiGX.*',
+               'GeoExt.*',
+               'momo.*'
```

Comment:
Mhm… will this result in really all files being in the build? If so I am against this. If not, go ahead. Still feels messy as I think we shoudl declare our dependencies where we actually depend on them being present (i.e. in class files).

If you have a strong opinion on this go ahead and merge regardless of my headaches.